### DISCUSSION TITLE
The parameter like "-m" too light. How should I modify the related configs?

### DISCUSSION THREAD

**Discussion by lentikr** [wezterm/wezterm#3506] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/95539012/231325766-c4c29d8a-b316-4a31-bfe9-ed237bf8a367.png)

**Comment by nekowinston** [comment, marked as answer]
That's an issue with your colorscheme and opacity.
If you run
```bash
for i in {0..15} ; do
    printf "\x1b[38;5;${i}m%3d " "${i}"
    if (( $i == 15 )) || (( $i > 15 )) && (( ($i-15) % 12 == 0 )); then
        echo;
    fi
done
```
in bash or zsh, you'll get an overview like this
![2023-04-12-044939@2x](https://user-images.githubusercontent.com/79978224/231335502-7df7d6a8-28fd-415b-9ba8-5d043487e5ef.png)

0-7 are part of the `ansi` table, 8-15 are part of the `brights` table [here](https://wezfurlong.org/wezterm/config/appearance.html#defining-your-own-colors).

An easy solution would be to switch your colorscheme.

To [override](https://wezfurlong.org/wezterm/config/appearance.html#defining-a-color-scheme-in-your-weztermlua) certain colors, use this:
```lua
c = wezterm.config_builder()

local custom = wezterm.color.get_builtin_schemes()["Catppuccin Mocha"]
custom.ansi[3] = "#ff69b4"
custom.brights[3] = "#ff69b4"
c.color_schemes = {
  ["Custom"] = custom,
}
c.color_scheme = "Custom"
```
Remember that Lua begins to index its tables with 1, so the first entry is `ansi[1]` instead of `ansi[0]`.
